src/main/java/de/tum/in/www1/artemis/service/GroupNotificationService.java

```diff
@@ -70,18 +70,18 @@ public void notifyStudentGroupAboutQuizExerciseStart(QuizExercise quizExercise)
     }
 
     /**
-     * Notify student groups about an exercise update.
+     * Notify all groups but tutors about an exercise update.
```

Comment:
It might make sense to also note down this rationale here: (Why not also tutors? -> Tutors will read the exercise during the assessment. It is only critical for students to know about changes as quickly as possible and Instructors/Editors to verify the changes.) so later readers see it in the code and don't have to look up the PR

Reply:
Sure, good point. Thanks 👍